test/Microsoft.Azure.WebJobs.Host.EndToEndTests/Scale/ScaleHostEndToEndTests.cs

```diff
@@ -20,6 +24,9 @@
 using Xunit;
 using static Microsoft.Azure.WebJobs.Host.EndToEndTests.Scale.ScaleHostEndToEndTests;
 
+[assembly: WebJobsStartup(typeof(TestExtensionA), "testExtensionATrigger")]
```

Comment:
Are you proposing we'd update all the existing extensions this way with an Name ctor param? I think we should use all the existing assembly attributes declared by extensions as is. If not, recommend using in this test what we expect extensions to use. I.e. I think all extensions are currently accepting the default value for Name.

Reply:
I add `ScaleHostBuilderContext.GetTriggersMetadata(string type)` instead.